restore/remote.go

```diff
@@ -38,8 +38,25 @@ func VerifyBackupDirectoriesExistOnAllHosts() {
 }
 
 func VerifyBackupFileCountOnSegments() {
+	origSize, destSize, isResizeRestore := GetResizeClusterInfo()
+	cmdMap := make(map[int][]int, destSize)
+	for i := 0; i < origSize; i++ {
+		cmdMap[i%destSize] = append(cmdMap[i%destSize], []int{i}...)
+	}
+	for i := origSize; i < destSize; i++ {
+		cmdMap[i] = append(cmdMap[i], []int{i}...)
+	}
+
+	GetDirsForContent := func(i int) string {
+		dirs := ""
+		for _, e := range cmdMap[i] {
```

Comment:
Not critical, but can we do the same without intermediate map?
___
Not tested, but something like that:
```
for contentID < origSize || contentID < destSize {
	dirs += " " + fmt.Sprintf("%s", globalFPInfo.GetDirForContent(contentID))
	contentID += destSize
}
```
And of course do this with the comment. It took a time for me to understand what the whole block is doing. And I still don't understand why we use non self-exceptionable e and i not for loop counter.

Reply:
map is used to distribute backup segments to restore segments (especially when origSize > destSize)

Reply:
how you compare contentID - segment identificator and origSize/destSize - quantity of segments?
what is mean addition quantity of segments to segment identificator?

Reply:
applied

Reply:
My request of comment was ignored.

Reply:
Comments were added